### PR TITLE
Create dummies for SignedEntities

### PR DESCRIPTION
**Pull request by dlachaumepalo**
## Content
This PR includes an update of `SignedEntity`. It adds a dummy for each `SignedEntity` type that simplifies code in tests.

## Pre-submit checklist

- Branch
  - [ ] Tests are provided (if possible)
  - [x] Crates versions are updated (if relevant)
  - [x] Commit sequence broadly makes sense
  - [x] Key commits have useful messages
- PR
  - [x] No clippy warnings in the CI
  - [x] Self-reviewed the diff
  - [x] Useful pull request description
  - [x] Reviewer requested
- Documentation
  - [ ] Update README file (if relevant)
  - [ ] Update documentation website (if relevant)
  - [ ] Add dev blog post (if relevant)

### CODE REVIEW

File: mithril-common/src/entities/signed_entity.rs [hunk 1]
```diff
@@ -24,3 +27,37 @@ where
     /// Date and time when the signed_entity was created
     pub created_at: DateTime<Utc>,
 }
+
+impl SignedEntity<Snapshot> {
+    cfg_test_tools! {
+    /// Create a dummy [SignedEntity] for [Snapshot] entity
+    pub fn dummy() -> Self {
+        SignedEntity {
+            signed_entity_id: "signed-entity-id-123".to_string(),
```
Comment (jpraynaud) [inline mithril-common/src/entities/signed_entity.rs:36]: For readability:
```suggestion
            signed_entity_id: "snapshot-123".to_string(),
```

File: mithril-common/src/entities/signed_entity.rs [hunk 2]
```diff
@@ -24,3 +27,37 @@ where
...
+    }
+    }
+}
+
+impl SignedEntity<MithrilStakeDistribution> {
+    cfg_test_tools! {
+        /// Create a dummy [SignedEntity] for [MithrilStakeDistribution] entity
+        pub fn dummy() -> Self {
+            SignedEntity {
+                signed_entity_id: "signed-entity-id-123".to_string(),
```
Comment (jpraynaud) [inline mithril-common/src/entities/signed_entity.rs:53]: Also:
```suggestion
                signed_entity_id: "mithril-stake-distribution-123".to_string(),
```

**Review by jpraynaud — COMMENTED** [review]
Few comments,  otherwise LGTM

**Review by jpraynaud — APPROVED** [review]
LGTM :+1:

**Review by ghubertpalo — APPROVED** [review]

**Pull request merged**